tests/e2e-evm/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "lib": ["ES2023"],
+    "target": "ES2022",
+    "module": "node16",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "resolveJsonModule": true
+  }
```

Comment:
may want to add `"noFallthroughCasesInSwitch": true,` to make sure you always have a default for switch cases.

Reply:
maybe `"isolatedModules": true,` too?
https://www.typescriptlang.org/tsconfig/#isolatedModules

Reply:
Added in future commit